addons/mail/static/src/core/common/thread.js

```diff
@@ -169,7 +170,7 @@ export class Thread extends Component {
                         : height - pt - pb - (this.env.inChatter?.aside ? 75 : 0)
                 }px)`;
             },
-            () => [this.jumpPresentRef.el, this.viewportEl]
+            () => [this.jumpPresentRef.el, this.viewportEl, this.props.activeAction]
```

Comment:
This technique works when toggling action panel but not when resizing it.
<img width="1284" alt="Screenshot 2024-11-05 at 21 29 56" src="https://github.com/user-attachments/assets/18447a34-2ee6-4c7a-8ab8-35507110a055">
I'd suggest to use a resize observer on thread.root.el so that when its size changes, it recomputes the position of the jump button.